packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/ios/challengeintegrationsteps.md

```diff
@@ -0,0 +1,43 @@
+### 1 - 4: Sign-in and Select Authenticator
+The challenge flow follows the same first 3-4 steps as the [enrollment flow](/docs/guides/authenticators-google-authenticator/ios/main/#integrate-sdk-for-authenticator-enrollment):
+
+* Display the initial sign-in UI
+* Start the sign-on flow
+* Authenticate the user credentials
+* Display a list of possible authenticator factors
```

Comment:
Good catch of clumsy wording. It actually is the list of authenticators in which the user can enroll rather than ones that have already enrolled in. How about:

> Display the list of authenticator choices